Shelfie.Identity/Shelfie.Identity.Api/Models/LoginModel.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Shelfie.Identity.Api.Models;
+
+public class LoginModel
+{
+    [Required(ErrorMessage = "User Name is required")]
+    public string? Username { get; set; }
+
+    [Required(ErrorMessage = "Password is required")]
```

Comment:
Please use fluent validation as it can be unit tested and it is easier to maintain